django/thunderstore/core/settings.py

```diff
@@ -279,6 +279,7 @@ def load_db_certs():
 
 # Session cookie used by React components during Django React transition
 SESSION_COOKIE_HTTPONLY = False
+SESSION_COOKIE_DOMAIN = ".%s" % env.str("SERVER_NAME")
```

Comment:
The `SERVER_NAME` is basically obsolete by now (only used in 2 places which will also be phased out eventually), so it's not a good idea to rely on it.

I'd recommend declaring a new environment variable (e.g. just `SESSION_COOKIE_DOMAIN`) and reading the value from it directly, or if not present, default to `None` as is the Django default. Make sure the default (not set) also evaluates to `None` instead of `"None"` with env.str, I'm unsure if it converts it or not. Also don't forget to document it to the `README.md` somewhere where it makes sense.

Reply:
Added the env variable and made a check simple len check and "None" string check for it, just in case. Also added a note on it in the Readme and a disclaimer of the problem with Django that happens; when an URL without ending is given to it, Django throws the Cookie to some limbo never to be found again. Not 100% sure what happens there, but found no documentation or bug reports about it.